interfaces.yaml

```diff
@@ -0,0 +1,37 @@
+oc-interface-paths:
+  # /interfaces/interface/config:
```

Comment:
Why commented?  Do we skip test for `config` part?

Reply:
Well, yes. Most devices do not send that. But perhaps they should and we can add that anyway.